src/helper/helper.js

```diff
@@ -123,8 +150,21 @@ class DataLayerHelper {
      */
     this.abstractModelInterface_ = buildAbstractModelInterface_(this);
 
+    if (processNow) {
+      this.process();
+    }
+  }
+
+  /**
+   * Processes the current dataLayer and registers the set command.
+   * The helper will not respond to pushes to the dataLayer until
+   * this method has been executed. Unless the processNow argument is
+   * intentionally set to false via the constructor, this method will
+   * always execute at construction time.
+   */
+  process() {
```

Comment:
You could also add a error message here if process has already been called.

Reply:
+1, probably worth a warning at least.

Reply:
Added the warning with a new internal marker for checking if the DLH has been processed before!